include/chopper/layout/configuration.hpp

```diff
@@ -39,6 +44,30 @@ struct configuration
     bool force_all_binnings{false};
     bool output_statistics{false};
     bool debug{false};
+
+private:
+    friend class cereal::access;
+
+    template <class Archive>
+    void serialize( Archive & ar )
+    {
+      ar(CEREAL_NVP(data_file.string()));
+      ar(CEREAL_NVP(output_filename.string()));
+      ar(CEREAL_NVP(hll_dir.string()));
+      ar(CEREAL_NVP(t_max));
+    //   ar(CEREAL_NVP(aggregate_by_column));
```

Comment:
Currently not supported, right?

Since we write json, the order is not that important. If we wrote binary, we cannot insert a variable later (just append).

We might just add it, it doesn't hurt?

Reply:
I wouldn't add it now in case we don't manage to implement it before a release of the paper and then the user wonders what it is when lookin at it.